src/lib/plugin/manifest.ts

```diff
@@ -0,0 +1,111 @@
+import { isLeft, toError, tryCatch } from 'fp-ts/lib/Either'
+import * as fs from 'fs-jetpack'
+import * as Path from 'path'
+import { PackageJson } from 'type-fest'
+import { fatal } from '../process'
+import { getPackageJsonMain } from '../utils'
+import { Dimension, DimensionEntrypointLocation, Manifest, Plugin, ValidatedPackageJson } from './types'
+
+/**
```

Comment:
Did think about moving Manifest type to this module... But its late and don't want to open can of worms (mostly re adding more circ deps).

Reply:
I generally find it pretty convenient to have types centralized in a single file. When they're scattered across multiple files it's always a maze to know where they are